gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ideaVersion = IC-15.0.6
+ideaVersion = IU-162.1447.7
```

Comment:
Makes sense. So locally we'll have to update the target IJ to older versions when testing. Also even though travis builds with java 8 we should keep the jdk target locally to 7.